cmd/monaco/deploy/logging.go

```diff
@@ -27,10 +27,7 @@ func logProjectsInfo(projects []project.Project) {
 	for _, p := range projects {
 		log.Info("  - %s", p)
 	}
-
-	if log.DebugEnabled() {
```

Comment:
yes, I've deleted this check.
if you think this is absolutely needed I would probably need to add such a method to get to the log level to the logger interface.... also it feels not 100% correct to base this check on the logger configuration

Reply:
IIRC I was the one to add it, and I can generally live with that. 
But I added the guard as it's purely debug logging and the operation is somewhat heavy, especially for large config sets (nested loop O(n*m*k), n..projects, m...environments, k..configs).

Open to discussing the necessity/practice of debug logging guards, but in general, I think it's a good practice and might be worth re-adding to our logging interface. 
(btw: exposing the level/whether a level below INFO is active via a method is what typical Java logging frameworks do, that's probably why it does feel like the right way to expose the information me.)

Reply:
I will add it :)